pkg/commands/product/bot_management/disable.go

```diff
@@ -0,0 +1,77 @@
+package bot_management
+
+import (
+	"io"
+
+	"github.com/fastly/go-fastly/v9/fastly"
+	"github.com/fastly/go-fastly/v9/fastly/products/bot_management"
+
+	"github.com/fastly/cli/pkg/api"
+	"github.com/fastly/cli/pkg/argparser"
+	"github.com/fastly/cli/pkg/global"
+	"github.com/fastly/cli/pkg/manifest"
+	"github.com/fastly/cli/pkg/text"
+)
+
+// DisableFn is a dependency-injection point for unit tests to provide
+// a mock implementation of the API operation.
+var DisableFn = func(client api.Interface, serviceID string) error {
+	return bot_management.Disable(client.(*fastly.Client), serviceID)
+}
+
+// DisableCommand calls the Fastly API to disable the product.
+type DisableCommand struct {
+	argparser.Base
+	Manifest manifest.Data
+
+	serviceName argparser.OptionalServiceNameID
+}
+
+// NewDisableCommand returns a usable command registered under the parent.
+func NewDisableCommand(parent argparser.Registerer, g *global.Data) *DisableCommand {
+	c := DisableCommand{
+		Base: argparser.Base{
+			Globals: g,
+		},
+	}
+	c.CmdClause = parent.Command("disable", "Disable the "+bot_management.ProductName+" product")
```

Comment:
At first, I was rather confused until I realized that `bot_management.ProductName` refers to `github.com/fastly/go-fastly/v9/fastly/products/bot_management.ProductName` and not to something in this package. Same for the rest of the `bot_management.Thing`s in `pkg/commands/product/bot_management/*.go` files.

Reply:
I was confused by that when I started working in Go; if you're in package A, and you've also imported a package named A, then qualified references like `A.foo` *always* refer to the imported package and never to the 'local' package. Now that I've gotten used to it I'm not bothered by it, but this sort of thing is what prompted the desire for import aliases in the other project :-)

Reply:
Might I suggest an alias is probably appropriate here? `bot` or `botapi` could work

Reply:
Try it and see what feels best, I guess!